fetch.bs

```diff
@@ -728,8 +728,9 @@ explicitly set <a for=/>request</a>'s
 this flag set are subject to additional processing requirements.
 
 <p>A <a for=/>request</a> has an associated
-<dfn export>skip-service-worker flag</dfn>. Unless stated otherwise it is
-unset.
+<dfn for=request export>service-workers mode</dfn>, that is "<code>all</code>",
```

Comment:
Would it make sense to add some non-normative text somewhere to explain what this mode actually means? Of course you can figure that out by reading the algorithms, but I find the non-normative notes that many of the requests attributes have to be quite helpful in quickly figuring out what a particular thing is for.